src/client.cr

```diff
@@ -54,11 +59,20 @@ module Crudris
 
     def close
       @ws.not_nil!.close
+	  Log.info { "Connection closed to gateway." }
       exit(0)
     end
 
     def create_message(content : String)
-      headers = HTTP::Headers{
+      if content.nil?
+		raise "Content cannot be nil."
+	  elsif content.empty?
+		raise "Content cannot be empty."
+	  elsif content.size > 2048 || content.size < 1
```

Comment:
Dynamic for the instance, alternatively you can just handle the response error.

Reply:
This was meant to be its own PR, I probably forgot about it when I pushed the above code. But thank you for the suggestion, it will be on its own commit.